ada_feeding/ada_feeding/trees/move_to_mouth_tree.py

```diff
@@ -90,7 +90,7 @@ def __init__(
         allowed_face_distance: Tuple[float, float] = (0.4, 1.25),
         face_detection_msg_timeout: float = 5.0,
         face_detection_timeout: float = 2.5,
-        plan_distance_from_mouth: Tuple[float, float, float] = (0.025, 0.0, -0.01),
```

Comment:
Doe we expect this to ever not be 3D? When?

If not, we should keep it a Tuple to enforce the size.

Reply:
We always expect it to be length 3, but the issue is that ROS2 params cannot be of type tuple, only type list. And Python's typing interface doesn't currently allow us to specify a length for Lists. 

However, I found [this discussion](https://github.com/python/mypy/issues/8441#issuecomment-1325434751) in the `mypy` library, which is a static type checker for Python. So I'll change it to `Annotated[Sequence[float], 3]` so that future Python type checkers can verify it.